packages/cli/src/tools/executeWinCommand.ts

```diff
@@ -0,0 +1,10 @@
+import execa from 'execa';
+
+const executeCommand = async (command: string) => {
```

Comment:
I'll add some `jsdoc` comments where missing if things look ok

Reply:
sounds more like `executeShellCommand` isn't it? we normally don't use `{shell: true}` so we should make it explicit

Reply:
Done